news/2021-11-04-skinners-bulletin-august-2021.md

```diff
@@ -21,61 +21,65 @@ In August, the skinning forum has seen **39** skins being published, **20** of t
+Moving onto the gameplay interface, we see every aspect of the main menu reflected beautifully here too. However, this time gear wheels dominate the screen as if we are inside a typical clock. One of the most fabulous parts of the gameplay section is undoubtedly the pause menu, where menu options surround a cracked piece of clockwork, and the fail menu, a copy of the pause menu but with more cracks. The section pass/fail displays are another aspect of the skin dominated by gear wheels and Kronii herself, in addition to the skip button, which is centralised, making it look even more remarkable.
 
-Moving onto the gameplay interface, we see that every aspect of the main menu reflected beautifully here, too. However, this time gear wheels dominate the screen as if we are inside a typical clock. One of the most fabulous parts of the gameplay section is, without a doubt, the pause menu, where menu options surround a clockwork, and the fail menu, which is a cracked copy of the pause menu. Section pass/fail displays are another aspect of the skin dominated by gear wheels and Kronii herself, in addition to the skip button, which is centralized, making it look even more remarkable. In short, we highly recommend you take this lovely creation on a test ride! Thanks to the showcase video below, time constraints are no longer an issue:
+In short, we highly recommend you take this *timeless* creation on a test ride! Thanks to the showcase video below, time constraints are no longer an issue:
 
 <iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/NvVeoYrZMUo" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ---
 
-That said, it is time to conclude this issue of our bulletin!
+And with that, it is time to conclude this issue of our bulletin!
 
-We're sincerely sorry about the unexpected delay on our side; nonetheless, we're here and thrilled to bring you the loveliest skins of the month. Unfortunately, we regret to inform you that we will be publishing our bulletin on a seasonal basis from now on, meaning that the next issue of Skinners' Bulletin will be released in December. On the bright side, we will be showcasing more stats, more skins, and of course, more news & tips, all of which will be compiled from the past three months.
+We're sincerely sorry about the unexpected delay on our side, but nonetheless, we're here and thrilled to bring you the loveliest skins of the month. Unfortunately, we regret to inform you that we will be publishing our bulletin on a seasonal basis from now on, meaning that the next issue of Skinners' Bulletin will be released in December. On the bright side, we will be showcasing more stats, more skins, and of course, more news & tips, all of which will be compiled from the past three months.
```

Comment:
This paragraph has like, way too many ups and downs. I don't think the initial apology is necessary. Just leave it at saying that due to time/content/whatever constraints the posts will be quarterly (makes more sense than "seasonal").

Reply:
It appears that you've reviewed the outdated version of the document. Eph has already edited the paragraph the way you suggested.